eng/pipelines/dotnet-sqlclient-ci-core.yml

```diff
@@ -380,8 +380,8 @@ stages:
           operatingSystem: Mac
           configProperties:
             # config.json properties
-            TCPConnectionString: $(AZURE_DB_TCP_CONN_STRING)
-            NPConnectionString: $(AZURE_DB_NP_CONN_STRING)
+            TCPConnectionString: $(AZURE_DB_SP_TCP_CONN_STRING)
+            NPConnectionString: $(AZURE_DB_SP_NP_CONN_STRING)
```

Comment:
@David-Engel  out of curiosity what is `SP` in `AZURE_DB_SP_NP_CONN_STRING` ?

Reply:
Service Principal

However, its been removed now to meet security wave requirements. I temporarily added it to fix the pipeline while we figure out a way for Microsoft hosted machines to use managed identity.

Reply:
@saurabh500 SP = service principal. It's the app registration named "Microsoft.Data.SqlClient" in our subscription.